### ISSUE TITLE
Handle style change edge cases

### ISSUE THREAD

**Issue by emilgoldsmith** [ChessCom/browser-extension#92]
Like for example changing styles while in a different tab and then changing to chess.com, the same with reset button etc.

I seem to once in a while also not be able to change the styles before I refresh the page or some weird things, this could be something to do with the dev environment though, as sometimes when changing branches the browser has loaded an old background page. But should be checked that this doesn't happen either.

**Comment by martynchamberlin** [comment]
Agreed that there's some weird edge cases in this regard. I've definitely seen this too.
